PROVESFlightControllerReference/test/int/rtc_test.py

```
@@ -302,17 +308,22 @@ def test_05_rtc_alarm_set_and_trigger(fprime_test_api: IntegrationTestAPI, start
         Second=alarm_time.second,
     )
     alarm_time_data_str = json.dumps(alarm_time_data)
+    start: TimeType = TimeType().set_datetime(
+        datetime.now(), time_base=TimeType.TimeBase("TB_DONT_CARE")
```

Ruff (0.16.1)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)